app/scripts/lib/nonce-tracker.js

```diff
@@ -127,6 +121,17 @@ class NonceTracker {
     return highestNonce
   }
 
+  _getHighestContinuousFrom (txList, startPoint) {
+    const nonces = txList.map((txMeta) => parseInt(txMeta.txParams.nonce, 16))
```

Comment:
This does assume nonce on txParams is always hex 
Not sure if that's enforced anywhere